README.md

```diff
@@ -1 +1,77 @@
-# kubebuilder-plugin
\ No newline at end of file
+# Enable kubebuilder-plugin for operator-sdk
+
+
+To use kubebuilder-plugin for java operators we need to clone the operator-sdk repo. 
+
+### Updates in Operator-SDK go.mod
+
+- Add the kubebuilder plugin to `go.mod`
+
+```
+github.com/java-operator-sdk/kubebuilder-plugin v0.0.0-20210225171707-e42ea87455e3
+```
+
+- Replace the kubebuilder-plugin path in go-mod pointing to the local dir of your kube-builder repo. Example.
+
+```
+github.com/java-operator-sdk/kubebuilder-plugin => /Users/sushah/go/src/github.com/sujil02/kubebuilder-plugin
+```
+
+### Updates in Operator-SDK `internal/cmd/operator-sdk/cli/cli.go`
+
+- Add the java-operator-sdk import
+
+```
+javav1 "github.com/java-operator-sdk/kubebuilder-plugin/pkg/java/v1"
+```
+
+- Introduce the java bundle in `GetPluginsCLIAndRoot()` method. 
+```
+javaBundle, _ := plugin.NewBundle("quarkus"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
+		&javav1.Plugin{},
+	)
+```
+
+- Add the created javaBundle to the `cli.New`
+
+```
+    cli.WithPlugins(
+			ansibleBundle,
+			gov2Bundle,
+			gov3Bundle,
+			helmBundle,
+			javaBundle,
```

Comment:
This could've been just `&javav1.Plugin{},` because Bundles *are* plugins that contains multiple plugins. And `WithPlugins` takes in a list of plugins.